cmake/mesh_dfu_metadata.cmake

```diff
@@ -41,3 +41,7 @@ function(mesh_dfu_metadata)
     COMMAND_EXPAND_LISTS
   )
 endfunction()
+
+if(SYSBUILD)
+  mesh_dfu_metadata()
```

Comment:
Does this work? I don't think `SYSBUILD` is set in sysbuild itself, only in images.

Reply:
Yes, it is set